sites/dentistryiq.com/server/styles/index.scss

```diff
@@ -22,17 +22,4 @@ $theme-site-header-breakpoints: (
   small-text-secondary: 598px
 );
```

Comment:
Remove `$theme-item-description-max-lines: 5;` from L15 (the core styles handle this now)